### ISSUE TITLE
Add support for --file option

### ISSUE THREAD

**Issue by sn3p** [sysgears/mochapack#20]
> #### [`--file <file/directory/glob>`](https://mochajs.org/#-file-filedirectoryglob)
> Explicitly include a test file to be loaded before other test files files. Multiple uses of --file are allowed, and will be loaded in order given.
> 
> Useful if you want to declare, for example, hooks to be run before every test across all other test files.
> 
> Files specified this way are not affected by --sort or --recursive.
> 
> Files specified in this way should contain one or more suites, tests or hooks. If this is not the case, consider --require instead.

**Comment by Jack-Barry** [comment]
@sn3p can you try this out with `2.0.0`? The CLI option for `--file` is sent to the `Mocha` instance as part of [`initMocha`](https://github.com/sysgears/mochapack/blob/51cfd136193af3ace4f9f419c319377969c82a73/src/runner/runnerUtils/initMocha/index.ts#L42) now.

**Comment by nicooga** [comment]
The file option is not working property for me, with mochapack 2.0.1. I've used this command:

~~~json
"test": "NODE_ENV=test mochapack --mode development --exit --require @babel/register --require babel-polyfill --file ./test/setup.js --recursive 'src/**/*.spec.js?(x)'",
"test-debug": "yarn test --inspect"
~~~

The `test/setup.js` file is being evaluated, but global variables set there are not present later. This is the same command I've used with mocha, except that before I used `mocha.opts` file to specify `file: 'test/setup.js'`.
